.dockerignore

```diff
@@ -0,0 +1,15 @@
+.venv
+.flake8
+.gitignore
+checklist.md
+.githab

```

Comment:
There is a typo in this line: `.githab` should be corrected to `.github` to ensure the `.github` directory is ignored by Docker.